opentreemap/importer/models/base.py

```diff
@@ -65,12 +66,10 @@ def status_summary(self):
             self.FAILED_FILE_VERIFICATION: "Invalid File Structure",
             self.CANCELED: "Canceled",
             self.VERIFICATION_ERROR: "Verification Error",
+            self.TASK_VANISHED: "Unknown Error",
```

Comment:
To be able to resume jobs I think we need:
```
self.VERIFICATION_TASK_VANISHED: "Failed During Verification",
self.CREATION_TASK_VANISHED: "Failed During Tree Creation",
```

... and to make the message right for species see https://github.com/OpenTreeMap/otm-core/blob/develop/opentreemap/importer/models/species.py#L50